app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,83 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    @tweet = Tweet.all
+    @user = User.current_user(session)
+    @user_all = User.all
+    if User.is_logged_in?(session)
+      erb :'/tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if User.is_logged_in?(session)
+      @user = User.current_user(session)
+      erb :'/tweets/create_tweet'
+    else
+      redirect to "/login"
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new", locals: {message: "Your tweet was empty."}
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => @user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.